modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -553,6 +555,98 @@ void cv::gimpl::GStreamingExecutor::setSource(GRunArgs &&ins)
         util::throw_error(std::logic_error("Only one video source is"
                                            " currently supported!"));
     }
+ 
+    // testing meta data is cv::GMatDesc{CV_8U,3,cv::Size{768,576}}
+    if (m_metas.empty()) {
+        // extract meta from first frame
+    } else {
+        //const auto& metas = m_metas;
+    }
+    
+    const auto& metas = m_metas;
+    
+    if (/*wasFinished*/true) {
```

Comment:
???

Reply:
This code is removed.
Cases with extract meta data and reshape will be implemented in future versions